### PR TITLE
added initial files

### PR DESCRIPTION
**Pull request by jkillz2020**

### CODE REVIEW

**Comment by jkillz2020** [comment]
@ncgonz It's ALIVE!!!!!!

**Comment by jkillz2020** [comment]
@ncgonz Just wanted to check in and see if you had comments on this?  I know you're busy looking at a bunch of quizzes, I'm just working from home today and thought I'd check.  Thanks!

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  var treeStuff = validateInputs();
+ growTree(treeStuff) }
+
+function growTree(treeObject){
+for (var i = 1; i <= treeObject.height; i++) {
+    newLine = "";
+    for (var j = 0; j <= treeObject.height-i-1; j++) {
+      newLine += " ";
+    }
+for (var j = 1; j<= 2*i - 1; j++) {
```
Comment (copypastedeveloper) [inline quiz.js]: indentation....... :)

**Comment by copypastedeveloper** [comment]
This all looks very nice, I only had one comment about the missing indentation on one line.  

The only other thing you might look into as an opportunity to challenge yourself is to look into how to solve this problem without nested for loops.  I'll give you a hint, strings have a repeat method on them.

**Comment by jkillz2020** [comment]
Thanks!  I'll definitely look at doing it with the repeat method.

**Comment by copypastedeveloper** [comment]
@jkillz2020 fix that indentation

**Comment by jkillz2020** [comment]
@ncgonz I think I fixed the indentation.

**Comment by copypastedeveloper** [comment]
looks good 👍

**Comment by jkillz2020** [comment]
Thanks!

**Pull request merged**